doc/Cookbook/Sims_phoSimGen.md

```diff
@@ -0,0 +1,134 @@
+
+To see the versions of a package available to your system, you can type
+```
+eups list my_package
+```
+which will show all of the versions of `my_package` available on your system, along with their corresponding `eups` tags.  If you want to see the directories in which these versions reside, use
+```
+eups list -v my_package
+```
+To see the versions of the LSST Simulations stack available to you, try
+```
+eups list sims_catUtils
+```
+`sims_catUtils` is the highest-level LSST package on which Twinkles depends.
+
+In order to use Twinkles, you must tell `eups` that Twinkles exists.  From the top level directory of `Twinkles` use
+```
+source setup/declare_eups.sh
+```
+to declare this package location and name to the `eups` database with a tag given by your user name. This step will not required to be repeated. 
+
+After you have done this, everytime you want to work with Twinkles in a new SHELL, you have to setup Twinkles. Once you have `eups` database loaded in that SHELL, you can source the
+script 
+```
+source setup/setup_twinkles.sh
+```
+if your sims stack has the tag `sims` (you can check by using `eups list -v sims_catUtils`).  Of these packages the one that is setup will have the tag followed by the word setup as following the tag of interest. If this tag is not 'sims' (for example, if you use a conda install, this tag is most likely 'current'), you can pass that tag to the setup script like this:
+```
+source setup/setup_twinkles.sh current
+```
+This script tells `eups` to setup Twinkles and a self-consistent copy of the LSST Simulations stack.  At this point, the directories of the Twinkles repositories become available as part of the `desc` namespace, and an environment variable `$TWINKLES_DIR` is exported to the SHELL.
+The code uses this variable.
+
+You will also need to tell python how to find your copy of OM10.  Go into the directory where you cloned the OM10 repository and follow the instructions in the README.md, namely
+```
+export OM10_DIR=$PWD
+export PYTHONPATH=$OM10_DIR:$PYTHONPATH
+```
+where `$PWD` refers to the home directory of your OM10 clone.
+
+### Connecting to the CatSim database
+
+For security reasons, the CatSim database exists behind a fairly restrictive firewall.  The two ways to access it are to get the IP address of your machine whitelisted for access through the firewall, or to establish an SSH tunnel through a communal account that has been created on one of the University of Washington computers.
+
+#### If you are whitelisted
+
+If your machine has been whitelisted, you must add the following entry to your `$HOME/.lsst/db-auth.paf` file:
+```
+database: {
+    authIfno: {
+        host: fatboy.phys.washington.edu
+        port: 1433
+        user: <shared username>
+        password: <shared password>
+    }
+}
+```
+
+You must then go into the the file `$LSST_HOME/sims_catUtils/config/db.py` and make change it so that
```

Comment:
It's not a big deal to change it, and I think it would be worthwhile (I was never comfortable with editing the contents of sims_catUtils, either).  I'll open a new issue and see if I can make this work.